web/packages/teleterm/src/ui/ConnectMyComputer/DocumentConnectMyComputerSetup/ProgressBar.tsx

```diff
@@ -151,3 +147,23 @@ function getPhaseSolidColor(theme: any): string {
   const alpha = decomposeColor(theme.colors.spotBackground[1]).values[3] || 0;
   return emphasize(theme.colors.levels.surface, alpha);
 }
+
+const Spinner = styled.div`
+  opacity: 1;
+  color: ${props => props.theme.colors.spotBackground[2]};
+  border: 3px solid ${props => props.theme.colors.success};
+  border-radius: 50%;
+  border-top: 3px solid ${props => props.theme.colors.spotBackground[0]};
+  width: 24px;
+  height: 24px;
+  position: absolute;
+  animation: spinner 4s linear infinite;
```

Comment:
Thoughts on making it `2s` as in the original `ProgressBar`? It's quite slow now.

Reply:
Same as above, I'd rather keep things consistent.

Reply:
But this component in cloud uses `Indicator` which has 2s animation (or [rather 1.5s now](https://github.com/gravitational/cloud/blob/4416a1f55b9824a45bc187e0ed4445a2dc119c96/web/packages/design/src/Indicator/Indicator.jsx#L75)) 🤔

Reply:
I don't think so, maybe they have two different components but I was referring to the one from the provisioning step.

https://github.com/gravitational/cloud/blob/4416a1f55b9824a45bc187e0ed4445a2dc119c96/web/packages/onboarding/src/StepProgress/Progress/Progress.tsx#L82
https://github.com/gravitational/cloud/blob/4416a1f55b9824a45bc187e0ed4445a2dc119c96/web/packages/onboarding/src/components/ProgressBar/ProgressBar.tsx#L128

Reply:
Ah ok, they changed it a week ago and no longer use Indicator.